docs/dev-upgrade.md

```diff
@@ -176,25 +176,25 @@ Edit these files:
 /opt/local/etc/pkgin/repositories.conf
 ```
 
-And change any instance of `2018Q4` to `2021Q4`, and any instance of
+And change any instance of `2021Q4` to `2024Q4`, and any instance of
 `pkgsrc.joyent.com` to `pkgsrc.smartos.org`.  There should be one
 instance in each file.  Here's is a pre-upgrade view:
 
 ```
 smartos-build(~)[0]% grep Q4 /opt/local/etc/pkg_install.conf 
-PKG_PATH=https://pkgsrc.joyent.com/packages/SmartOS/2018Q4/x86_64/All
+PKG_PATH=https://pkgsrc.joyent.com/packages/SmartOS/2021Q4/x86_64/All
```

Comment:
Is there a new URL that doesn't have `joyent` in it?

Reply:
I think it should be `pkgsrc.smartos.org`

Reply:
From my *old* zone...
```
[root@triton-build ~]# grep Q4 /opt/local/etc/pkg_install.conf
PKG_PATH=https://pkgsrc.smartos.org/packages/SmartOS/2021Q4/x86_64/All
```

Reply:
SHould get fixed any time now...